akka-actor/src/main/scala/akka/actor/Props.scala

```diff
@@ -192,6 +193,23 @@ final case class Props(deploy: Deploy, clazz: Class[_], args: immutable.Seq[Any]
    */
   def withDeploy(d: Deploy): Props = copy(deploy = d.withFallback(deploy))
 
+  /**
+   * Returns a new Props with the specified set of tags.
+   */
+  @varargs
+  def withTags(tag: String, additional: String*): Props = {
```

Comment:
maybe we should write it out as ActorTags, to make it clear that it's different from event tagging in EventSourceBehavior?

Reply:
Sounds good